tests/snmpsim/netscaler-sdx.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.5951.6
```

Comment:
Can the description be obfuscated in some parts? If I take the complete description string it would contain internal hostnames.